spanner/api/Spanner.Samples/WriteAndReadUsingDmlCoreAsync.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_dml_write_then_read]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class WriteAndReadUsingDmlCoreAsyncSample
+{
+    public async Task<int> WriteAndReadUsingDmlCoreAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        using var createDmlCmd = connection.CreateDmlCommand(@"INSERT Singers (SingerId, FirstName, LastName) VALUES (16, 'Timothy', 'Campbell')");
```

Comment:
The value in the [original sample is 11](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/7192e70f2f6e1ee8b8b5203b8768c89913d368db/spanner/api/Spanner/Program.cs#L1464), but if there's a good reason to make it 16 here, then I'm fine with it.